### DISCUSSION TITLE
activeClassName is not available on Link nor NavLink

### DISCUSSION THREAD

**Discussion by hazem-alabiad** [remix-run/react-router#8335] [General]
I am trying to set the `activeClassName` on either `NavLink` or `Link` but it is not available on both of them. Is it a bug? How to fix that, any suggestions?

**Comment by smashercosmo** [comment]
https://reactrouter.com/docs/en/v6/upgrading/v5#remove-activeclassname-and-activestyle-props-from-navlink-
